ansible_collections/arista/avd/molecule/eos_validate_state/inventory/group_vars/DC1_WAN.yml

```diff
@@ -16,7 +16,7 @@ wan_router: # dynamic_key: node_type
       enabled: true # TODO: AVD 4.8.0 Remove this once WAN HA is out of preview
   node_groups:
     - group: DC1-WAN
-      bgp_as: 65000
+      bgp_as: 65101
```

Comment:
hmmm why was this needed?

Reply:
Because of below error need make ASN change
```
TASK [arista.avd.eos_designs : Generate device configuration in structured format] ***
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ansible_collections.arista.avd.plugins.plugin_utils.errors.errors.AristaAvdError: Only iBGP is supported for WAN, the BGP AS 65101 on dc1-leaf1a is different from our own: 65000.
fatal: [dc1-wan1 -> localhost]: FAILED! => {"changed": false, "msg": "Only iBGP is supported for WAN, the BGP AS 65101 on dc1-leaf1a is different from our own: 65000."}
```

Reply:
ack thanks :)